src/molecules/header.tsx

```diff
@@ -1,25 +1,11 @@
 import React, { FunctionComponent } from 'react';
-import { graphql, useStaticQuery, Link } from 'gatsby';
-import Img from 'gatsby-image';
+import logo from '../images/Osqledaren_logga_svart.png';
 import styled from '../styles/styled';
 
 const Header: FunctionComponent = () => {
-  const data = useStaticQuery(graphql`
-    query {
-      logo: file(relativePath: { eq: "logo-black.png" }) {
-        childImageSharp {
-          fluid(maxWidth: 1000) {
-            ...GatsbyImageSharpFluid_noBase64
-          }
-        }
-      }
-    }
-  `);
   return (
     <HeaderWrapper>
-      <Link to="/">
-        <Logo fluid={data.logo.childImageSharp.fluid} />
-      </Link>
```

Comment:
Add the Link again around the img so you return to the first page when clicking on it.